dev/TEAL_opcodes.md

```diff
@@ -1059,7 +1089,37 @@ The call stack is separate from the data stack. Only `callsub` and `retsub` mani
 - pop the top instruction from the call stack and branch to it
 - Availability: v4
 
-The call stack is separate from the data stack. Only `callsub` and `retsub` manipulate it.
+If the current frame was prepared by `proto A R`, `retsub` will remove the 'A' arguments from the stack, move the `R` return values down, and pop any stack locations above the relocated return values.
```

Comment:
not necessary for specs, but for the dev doc I think we really need a nice diagram with what proto is doing.